src/content/en/fundamentals/media/mobile-web-video-playback.md

```diff
@@ -0,0 +1,611 @@
+      video.currentTime = 0;
+    });
+
+### Seek backward and forward
+
+Let's continue and add "seek backward" and "seek forward" buttons so that user
+can easily skip some content.
+
+<pre class="prettyprint lang-html">
+&lt;div id="videoContainer">
+  &lt;video id="video" src="file.mp4">&lt;/video&gt;
+  &lt;div id="videoControls">
+    &lt;button id="playPauseButton">&lt;/button>
+    <strong>&lt;button id="seekForwardButton">&lt;/button>
+    &lt;button id="seekBackwardButton">&lt;/button></strong>
+    &lt;div id="videoCurrentTime">&lt;/div>
+    &lt;div id="videoDuration">&lt;/div>
+    &lt;div id="videoProgressBar">&lt;/div>
+  &lt;/div>
+&lt;/div>
+</pre>
+
+    let skipTime = 10; // Time to skip in seconds
+
+    seekForwardButton.addEventListener('click', function(event) {
+      event.stopPropagation();
+      video.currentTime = Math.min(video.currentTime + skipTime, video.duration);
+    });
+
+    seekBackwardButton.addEventListener('click', function(event) {
+      event.stopPropagation();
+      video.currentTime = Math.max(video.currentTime - skipTime, 0);
+    });
+
+As before, rather than adjusting video styling in the `click` event listeners
+of these buttons, we'll use the fired `seeking` and `seeked` video events to
+adjust video brightness. My custom `seeking` CSS class is as simple as `filter:
+brightness(0);`.
+
+    video.addEventListener('seeking', function() {
+      video.classList.add('seeking');
+    });
+
+    video.addEventListener('seeked', function() {
+      video.classList.remove('seeking');
+    });
+
+Here's below what we have created so far. In the next section, we'll implement
+the fullscreen button.
+
+<video controls controlsList="nodownload" muted playsinline>
+  <source src="/web/fundamentals/media/videos/video-play-pause-seek.webm"
+          type="video/webm">
+  <source src="/web/fundamentals/media/videos/video-play-pause-seek.mp4"
+          type="video/mp4">
+</video>
+
+## Fullscreen
+
+Here we are going to take advantage of several Web APIs to create a perfect
+and seamless fullscreen experience. To see it in action, check out the
+[sample]{: .external }.
+
+Obviously, you don't have to use all of them. Just pick the ones that make
+sense to you and combine them to create your custom flow.
+
+<video controls controlsList="nodownload" muted playsinline>
+  <source src="/web/fundamentals/media/videos/perfect-fullscreen.webm"
+          type="video/webm">
+  <source src="/web/fundamentals/media/videos/perfect-fullscreen.mp4"
+          type="video/mp4">
+</video>
+
+### Prevent automatic fullscreen
+
+On iOS, `video` elements automagically enter fullscreen mode when media
+playback begins. As we're trying to tailor and control as much as possible our
+media experience across mobile browsers, I recommend you set the `playsinline`
+attribute of the `video` element to force it to play inline on iPhone and not
+enter fullscreen mode when playback begins.
```

Comment:
Maybe insist that there is no side effects on other browsers as they would either follow the attribute or by default play inline?